src/frontend/components/PictureInPictureLayer/index.tsx

```diff
@@ -34,13 +35,17 @@ const containerStyle = {
 interface PictureInPictureLayerProps {
   mainElement: ReactNode;
   secondElement?: ReactNode;
+  mainLayerContent?: ReactNode;
+  mainLayerProps?: Omit<LayerProps, 'target'>;
```

Comment:
I wonder if using a ref to access the container element wouldn't be better.
Like, expose a ref in PictureInPictureLayer and populate the value of this ref with the ref of the container element and maybe both element container as well.
What do you think ?

Reply:
I made a modification in a fixup commit. Tell me what do you think and then I will rework my commits if it's ok.

Reply:
Yep, I find it way better :)